src/app/common/messagedisplay.ts

```diff
@@ -36,20 +37,38 @@ export abstract class MessageDisplay {
     return this.rows.length;
   }
 
+  selectedRowIds(): number[] {
+    return Object.keys(this.selectedMessageIds).map((msgIndex) => this.findRowByMessageId(parseInt(msgIndex, 10)));
```

Comment:
Isn't `this.selectedMessageIds` a method? Did this mean to use this.`msgIdsSelected`?

This could be a good argument to give these more obvious names, right now we have attributes that look very similar to methods.

Reply:
Yes, oops (I renamed it mid fix)